### ISSUE TITLE
Hard to follow multiple effects

### ISSUE THREAD

**Issue by danj3000** [danj3000/ashteki#599]
Think about how to show what's going on

**Comment by 5H4Z** [comment]
I think at the moment we are erring on the side of providing more messages in the logs so that effects are not missed. This does occasionally lead to odd messages, like Law of Grace saying that it is removing a wound from a phoenixborn when there was no wound to remove in the first place, but across the card pool the messages are looking reasonably consistent even for multi-step effects.

**Comment by danj3000** [comment]
Agree things are much better for multi-step processes so will close.
There's still a query about multiple simultaneous effects, like meteor - there's a big change, lots of units lost, and you have to read up the chat to see what happened. I'll make a new issue to remind me to think about how we can make that easier to follow.
